compiler/src/dotty/tools/dotc/typer/Namer.scala

```diff
@@ -1454,7 +1454,7 @@ class Namer { typer: Typer =>
 
       addForwarders(selectors, Nil)
       val forwarders = avoidClashes(buf.toList)
-      exp.pushAttachment(ExportForwarders, forwarders)
+      exp.putAttachment(ExportForwarders, forwarders)
```

Comment:
On the other hand, it doesn't (at a glance) seem like this will ever give a different result, so maybe this is better?
```scala
      exp.getAttachment(ExportForwarders).getOrElse:
        addForwarders(selectors, Nil)
        val forwarders = avoidClashes(buf.toList)
        exp.putAttachment(ExportForwarders, forwarders)
        forwarders
```

Reply:
Yes, let's try that alternative. It seems better.